stubs/pydbus/@tests/stubtest_allowlist.txt

```diff
@@ -0,0 +1,18 @@
+pydbus
+pydbus._inspect3
+pydbus.auto_names
+pydbus.bus
+pydbus.bus_names
+pydbus.exitable
+pydbus.generic
+pydbus.identifier
+pydbus.method_call_context
+pydbus.proxy
+pydbus.proxy_method
+pydbus.proxy_property
+pydbus.proxy_signal
+pydbus.publication
+pydbus.registration
+pydbus.request_name
+pydbus.subscription
+pydbus.timeout
```

Comment:
Why are these modules added to stubtest_allowlist? This should not be necessary.

Reply:
The test said to do this when I did not have these here.